admin/backend/controllers/resources-controller.js

```diff
@@ -71,10 +71,16 @@ class ResourcesController extends BaseController {
   async findRecords({ query }) {
     this.view.perPage = 10
     this.view.total = await this.view.currentResource.count()
-    this.view.page = query.page || 1
+    const { page, sortBy, sortDirection } = query
+    this.view.page = page || 1
+    this.view.sort = {
+      sortBy: sortBy || '_id',
```

Comment:
in resource controller you cannot use raw field names like '_id' because in the future there will be other databases (which doesn't have _id). So here you have to iterate throught all model properties and find which property is an ID property (isId() method).

BUT
- i think that it should be filtered by default by first, visible in the list field :) - the simplest solution (this/view.currentResource.decorate().getListProperties() - first from that)

Reply:
I though that each document in a MongoDB collection has an _id - by default, but
{  $natural: 'asc' } query also returns the documents in the natural order

Reply:
done